Frends.Template/Frends.Echo.Execute/FrendsTaskMetadata.json

```diff
@@ -0,0 +1,7 @@
+﻿{
+	"Tasks": [
+		{
+			"TaskMethod": "Frends.Echo.Execute.Echo.Execute"
```

Comment:
Name looks sus, I haven't run this yet thought

Reply:
I named them differently because in some cases the ExecuteEcho Echo part got renamed as the Class.